resources/package.rb

```diff
@@ -4,6 +4,14 @@
 
 default_action :install
 
+def package_name
+  script = 'powershell "(Get-ItemProperty HKLM:\\\\Software\\\\Microsoft\\\\'\
+    'Windows\\\\CurrentVersion\\\\Uninstall\\\\* |  Select-Object DisplayName,'\
+    'Publisher |  Where-Object {$_.Publisher -like \'Alteryx\'}).DisplayName'
+  package_name = shell_out(script).stdout.strip
+  package_name.empty? ? 'Alteryx' : package_name
```

Comment:
This isn't a dealbreaker on the merge request, but we're going to want to be aware of conflicts that this could create in the future. For example, if Connect or Promote is installed, we don't want to grab the package name from the registry for those.